### PR TITLE
 [website] improve pulsar connector documentation

### PR DESCRIPTION
**Pull request by sijie**
### Changes

- Add documentation for each IO connector
- Add a tutorial how to run a built-in connector (end-to-end)
- How to develop a Pulsar IO connector

### CODE REVIEW

**Comment by sijie** [comment]
retest this please

**Comment by sijie** [comment]
rerun java8 tests

File: site2/docs/getting-started-standalone.md [hunk 1]
```diff
@@ -58,7 +58,51 @@ Directory | Contains
...
 
 
+## Installing Builtin Connectors
 
+Since release `2.1.0-incubating`, Pulsar releases a separate binary distribution, containing all the `builtin` connectors.
+If you would like to enable those `builtin` connectors, you can download the connectors tarball release in one of the following ways:
+
+* by clicking the link below and downloading the release from an Apache mirror:
+
+  * <a href="pulsar:connector_release_url" download>Pulsar IO Connectors {{pulsar:version}} release</a>
```
Comment (jerrypeng) [inline site2/docs/getting-started-standalone.md]: You will update the url with the correct link after the binaries are built?

Comment (sijie) [reply site2/docs/getting-started-standalone.md]: pulsar:connector_release_url are templates. so it will be replaced when building the website.

Comment (jerrypeng) [reply site2/docs/getting-started-standalone.md]: gotcha

File: site2/docs/io-develop.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+If you plan to package and distribute your connector for others to use, you are obligated to
+properly license and copyright your own code and to adhere to the licensing and copyrights of
+all libraries your code uses and that you include in your distribution. If you are using the
+approach described in ["Creating a NAR package"](#creating-a-nar-package), the NAR plugin will
+automatically create a `DEPENDENCIES` file in the generated NAR package, including the proper
+licensing and copyrights of all libraries of your connector.
+
+#### Creating a NAR package
+
+The most easiest approach to packaging a Pulsar IO connector is to create a NAR package using
```
Comment (jerrypeng) [inline site2/docs/io-develop.md]: delete "most"

File: site2/docs/io-managing.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+## Running Connectors
+
+Pulsar connectors can be managed using the [`source`](reference-pulsar-admin.md#source) and [`sink`](reference-pulsar-admin.md#sink) commands of the [`pulsar-admin`](reference-pulsar-admin.md) CLI tool.
+
+### Running sources
+
+You can use the [`create`](reference-pulsar-admin.md#source-create)
+
+You can submit a sink to be run in an existing Pulsar cluster using a command of this form:
```
Comment (jerrypeng) [inline site2/docs/io-managing.md]: Think should be for sources

File: site2/docs/io-managing.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+```bash
+$ ./bin/pulsar-admin sink create --className  <classname> --jar <jar-location> --tenant test --namespace <namespace> --name <sink-name> --inputs <input-topics>
```
Comment (jerrypeng) [inline site2/docs/io-managing.md]: should be source create

File: site2/docs/io-managing.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+The example yaml basically tells Pulsar which Cassandra cluster to connect, what is the `keyspace` and `columnFamily` to be used in Cassandra for collecting data,
+and how to map a Pulsar message into Cassandra table key and columns.
+
+For details, consult the documentation for [individual connectors](io-overview.md#working-with-connectors).
+
+## Running Connectors
+
+Pulsar connectors can be managed using the [`source`](reference-pulsar-admin.md#source) and [`sink`](reference-pulsar-admin.md#sink) commands of the [`pulsar-admin`](reference-pulsar-admin.md) CLI tool.
+
+### Running sources
```
Comment (jerrypeng) [inline site2/docs/io-managing.md]: For built-in sources and sinks we should probably also have an example to how to submit them.  In another page you go over in detail how to submit the cassandra connector but perhaps we can also explain how to run a built-in connector without submitting a jar and just by specifying the connector type

**Comment by jerrypeng** [comment]
Just a few comments. The documentation looks very good overall

**Comment by sijie** [comment]
@jerrypeng I have addressed all your comments.

**Comment by sijie** [comment]
retest this please

**Pull request merged**